packages/grid/x-data-grid/src/utils/utils.ts

```diff
@@ -170,3 +170,20 @@ export function isDeepEqual(a: any, b: any) {
   // eslint-disable-next-line no-self-compare
   return a !== a && b !== b;
 }
+
+// Pseudo random number. See https://stackoverflow.com/a/47593316
+function mulberry32(a: number): () => number {
+  return () => {
+    /* eslint-disable */
+    let t = (a += 0x6d2b79f5);
+    t = Math.imul(t ^ (t >>> 15), t | 1);
+    t ^= t + Math.imul(t ^ (t >>> 7), t | 61);
+    return ((t ^ (t >>> 14)) >>> 0) / 4294967296;
+    /* eslint-enable */
+  };
+}
+
+export function randomNumberBetween(seed: number, min: number, max: number): () => number {
+  const random = mulberry32(seed);
```

Comment:
With just math random on each scroll, the length of the skeleton cells is different. Passing a specific seed makes it stable, otherwise, it looks a bit buggy.

Reply:
For now, we can keep it in the helper's file, yes.